ui/plugins/controller.js

```diff
@@ -88,7 +93,11 @@ treeherder.controller('PluginCtrl', [
                     $scope.eta_abs = Math.abs($scope.job.get_current_eta());
                     $scope.typical_eta = $scope.job.get_typical_eta();
                     $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision
-
+                    $scope.job_ids = results[4];
+                    console.log(results);
+                    _.forEach($scope.job_ids, function(job) {
+                        console.log(job[0].value);
+                    });
```

Comment:
So looking at the json I see at least signature `32206df28ab3b52e15d26f73178ee78e2af7e760` which corresponds to ts_paint. 

So if you're not seeing it in your output, there must be a bug in the way you're formatting things. Let me know if you need help tracking this down, but I suspect it's a pretty simple problem. :)